packages/ember-data/lib/serializers/json-api-serializer.js

```diff
@@ -58,12 +58,12 @@ var dasherize = Ember.String.dasherize;
               "data": [
                 {
                   "id": "3",
-                  "type": "players"
+                  "type": "player"
```

Comment:
Store.push expects singular, JSON API serializer expects plural? I thought it is unified. Should I revert changes to json-api-serializer?